client/src/app/page/filter/datepicker-filter/datepicker-filter.component.ts

```diff
@@ -34,6 +36,12 @@ export class DatepickerFilterComponent implements OnInit {
     this.choiseReadyFlag = 0;
   }
 
+  private get defaultTitle(): string {
+    this._defaultTitle = this.filterItem.options.filter((item: FilterOptions) => item.value === -1)[0].name;
+
+    return this._defaultTitle;
```

Comment:
you shouldn't use underscore _defaultTitle

just return this.filterItem.options.filter((item: FilterOptions) => item.value === -1)[0].name;